nipy/algorithms/statistics/formula/tests/test_formula.py

```diff
@@ -18,7 +18,7 @@
 from ..formulae import terms, Term
 from nipy.utils import VisibleDeprecationWarning
 
-from nibabel.py3k import asbytes
+from numpy.compat.py3k import asbytes
```

Comment:
IIRC numpy is deprecating this as well. We should just change `asbytes(s)` to `s.encode()`.

Reply:
Seems logical. Adjusted.